### DISCUSSION TITLE
Hive catlog to JDBC catalog

### DISCUSSION THREAD

**Discussion by deril-27051998** [zsvoboda/ngods-stocks#20] [General]
Why you replaced hive catalog with JDBC catalog? any specific reason? i am also trying to connect hive catlaog from trino but not able to do it!

**Comment by zsvoboda** [comment]
I switched to the JDBC catalog because of the Hive-based metastore footprint (memory, download size, etc.).  Trino's JDBC catalog config is in this config file https://github.com/zsvoboda/ngods-stocks/blob/main/trino/conf/trino/catalog/warehouse.properties . Do you have something similar in your Trino config?

**Reply by deril-27051998** [reply]
Yeah tried via Hive catalog i am not able to get the tables to Trino . when I tired JDBC as what u worked i can be able to get the data to Trino.
So just checking did you faced any problems when you worked on Hive catalog?

**Comment by zsvoboda** [comment]
No, actually, there is a historical version of this repo that uses the
Hive-based metastore. The JDBC catalog has been introduced in this commit
https://github.com/zsvoboda/ngods-stocks/commit/d3eb4a1c24be2892ecaf22ed7317995d81986aa8
So the version of this repo before this commit is an example of working
Hive metastore integration.

ZD

On Wed, Apr 26, 2023 at 1:38 PM deril-27051998 ***@***.***>
wrote:

> Yeah tried via Hive catalog i am not able to get the tables to Trino .
> when I tired JDBC as what u worked i can be able to get the data to Trino.
> So just checking did you faced any problems when you worked on Hive
> catalog?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zsvoboda/ngods-stocks/discussions/20#discussioncomment-5730443>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQWAWDVJCPOTL3SJ537OLXDECMDANCNFSM6AAAAAAXJ7MOAQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
-- 
Zdenek Svoboda
***@***.*** ***@***.***>
m: +420 777 640 979

**Reply by deril-27051998** [reply]
Oh ok great! thanks! thanks for the great work ! it help me lot.
